### ISSUE TITLE
Cargo is on all Ships 0

### ISSUE THREAD

**Issue by Lucas-Gruber** [StarCitizenWiki/scunpacked-data#5]
Hello,
if you look at the Caterpillar data, for example, the cargo value is 0, and I think that applies to all ships.

**Comment by octfx** [comment]
Yes that is (sadly) correct, cargo capacity and thrust data is currently broken in the json data.
It works on the Wiki API though, as there we calculate the attributes directly from the hardpoints:
[Example for SCU](https://github.com/StarCitizenWiki/API/blob/a37f990fb4b2f7283aa7ad1aa9f3c317db51592c/app/Models/SC/Vehicle/Vehicle.php#L274)

I did not have the time yet to port this over to scunpacked :/

**Comment by octfx** [comment]
I've ported the logic over in 83e8331acf08dbfaec296bbe6d3cd04f6a952564
Would you mind confirming if this is correct?

**Comment by Lucas-Gruber** [comment]
I don't think its correct. For example if you go to the Constellation Taurus it has Cargo of 6 SCU instead of 174.

Sorry for the really late answer but i didn't had any time because of my job.

**Comment by octfx** [comment]
(Hopefully) fixed in upcoming release
